app/services/resources/resources.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Services
+  module Resources
```

Comment:
Might be cool to add some rspec tests for these services, but if they are already covered by the Postman tests, that's cool.